src/utils/url-utils.ts

```diff
@@ -4,95 +4,104 @@ declare global {
     }
 }
 
+export enum UrlFormat {
+    LOCAL,      // localhost, github pages etc
+    ADEO,       // app.adeo.no / modapp.adeo.no
+    NAIS,       // nais.preprod.local / nais.adeo.no
+    DEV_ADEO,   // dev.adeo.no
+    NAV_NO      // dev.intern.nav.no / intern.av.no
+}
+
 interface UrlEnvironment {
     environment: string;
     envclass: string;
-    isNaisUrl: boolean;
+    urlformat: UrlFormat;
 }
 
 interface UrlRule {
     regExp: RegExp;
+
     ifMatch(match: RegExpExecArray): UrlEnvironment;
 }
 
 const urlRules: Array<UrlRule> = [
     {
         regExp: /localhost/,
-        ifMatch: () => ({ environment: 'local', isNaisUrl: false, envclass: 'local' })
+        ifMatch: () => ({environment: 'local', envclass: 'local', urlformat: UrlFormat.LOCAL})
     },
     {
         regExp: /navikt\.github\.io/,
-        ifMatch: () => ({ environment: 'local', isNaisUrl: false, envclass: 'local' })
+        ifMatch: () => ({environment: 'local', envclass: 'local', urlformat: UrlFormat.LOCAL})
     },
     {
         regExp: /\.herokuapp\.com/,
-        ifMatch: () => ({ environment: 'local', isNaisUrl: false, envclass: 'local' })
+        ifMatch: () => ({environment: 'local', envclass: 'local', urlformat: UrlFormat.LOCAL})
     },
     {
         regExp: /\.labs\.nais\.io/,
-        ifMatch: () => ({ environment: 'local', isNaisUrl: true, envclass: 'local' })
+        ifMatch: () => ({environment: 'local', envclass: 'local', urlformat: UrlFormat.LOCAL})
```

Comment:
Hva er egentlig `labs.nais`?

Reply:
Det er tilsvarende denne: https://navlab.no/ om jeg ikke tar helt feil. 
Men tldr; tenk ett test-miljø hvor vi kunne ha deployet en testversjon for å vise frem noe til resten av nav.